src/pages/financial.tsx

```diff
@@ -1,7 +1,101 @@
+import { useState } from 'react';
 import { NextPageWithLayout } from './_app';
+import { Button } from '../components/atoms/Button';
+import Course from '../components/atoms/Course';
+
+//Course Interface
+interface course {
+  courseNumber: string;
+  courseName: string;
+  cost: number;
+  credit: number;
+};
+
+interface terms {
+  term: string;
+  courses: course[];
+}
+
+//Row Component
+const Row: ({ courseNumber, courseName, cost, credit }: course) => JSX.Element = ({ courseNumber, courseName, cost, credit }: course) => {
+
+  return (
+    <div className="flex items-center m-3 ">
+      <Course courseNumber={courseNumber} courseName={courseName} />
+      <p className="flex-1 text-center">${cost}</p>
+      <p className="flex-1 text-center">{credit}</p>
+    </div>
+  );
+};
+
 
 const FinancialPage: NextPageWithLayout = () => {
-  return <div>Financial Statements</div>;
+
+  const [courses, setCourses] = useState<course[]>([
+    { courseNumber: 'Seng 321', courseName: 'Requirements Engineering', cost: 200.99, credit: 1.5 },
+    { courseNumber: 'Seng 321', courseName: 'Requirements Engineering', cost: 200.99, credit: 1.5 },
+    { courseNumber: 'Seng 371', courseName: 'Software Evolution', cost: 200.99, credit: 1.5 }
+    ]);
+
+  const rows:JSX.Element[] = courses.map((course, index) => (
+    <Row key={index} courseNumber={course.courseNumber} courseName={course.courseName} cost={course.cost} credit={course.credit} />
+  ));
+
+  const totalCost:any = courses.reduce((total, current): any => {
+    if (total.cost && current.cost) {
+      return (total.cost + current.cost);
+    }
+    else {
+      return (Number(total) + current.cost);
+    }
+  });
+
+  const totalCredits:any = courses.reduce((total, current): any => {
+    if (total.credit) {
+      return (total.credit + current.credit);
+    }
+    else {
+      return (Number(total) + current.credit);
+    }
+  });
+
+  return (
```

Comment:
<img width="1351" alt="Screenshot 2023-03-05 at 12 33 28 PM" src="https://user-images.githubusercontent.com/31324580/222984426-3544829b-e509-437b-9554-bce2b4b8a1af.png">

the formatting is broken in the preview